akka-docs-dev/rst/stages-overview.rst

```diff
@@ -0,0 +1,134 @@
+buffer (Fail)                                                                                                                                      fails the stream instead of backpressuring when buffer is full                                                                  Upstream completes and buffered elements has been drained
+=====================  =========================================================================================================================   ==============================================================================================================================  =====================================================================================
+
+Nesting and flattening stages
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+These stages either take a stream and turn it into a stream of streams (nesting) or they take a stream that contains
+nested streams and turn them into a stream of elements instead (flattening).
+
+**It is currently not possible to build custom nesting or flattening stages**
+
+=====================  =========================================================================================================================   ==============================================================================================================================  =====================================================================================
+Stage                  Emits when                                                                                                                  Backpressures when                                                                                                              Completes when
+=====================  =========================================================================================================================   ==============================================================================================================================  =====================================================================================
+prefixAndTail          the configured number of prefix elements are available. Emits this prefix, and the rest as a substream                      downstream backpressures                                                                                                        After the specified number of prefix elements has been consumed or upstream completes
+groupBy                an element for which the grouping function returns a group that has not yet been created. Emits the new group               there is an element pending for a group whose substream backpressures                                                           Upstream completes [3]_
```

Comment:
That will not fit on the page, also it is uninteresting for 90% of the stages. I would rather put that detail in the scaladocs.